src/services/rfqm_service.ts

```diff
@@ -1075,6 +1076,8 @@ export class RfqmService {
                             actualTakerFillAmount: decodedLog.args.takerTokenFilledAmount.toString(),
                             decodedFillLog: JSON.stringify(decodedLog),
                         };
+                        // submissionsMap[r.transactionHash].priorityFee = r.response.priorityFee;
```

Comment:
this is commented about because the receipt doesn't include the effective gas price. 

you should just delete these.